### PR TITLE
Improve `README` file

### LINKED ISSUE #54: Improve the `README`
rubric={reasoning:5}

Expand on the README file to be a welcoming place for anyone coming to your project for the first time. For your project, your README should cater to at least two groups of people:

1. Those interested in using your dashboard
  - Include motivation behind your project and clearly explain what problem you are solving and why it is important.
  - You do not have to include detailed usage instructions, just high level what they can do in your dashboard and the deployed link.
[This is a good example](https://github.com/KirstieJane/STEMMRoleModels)
  - Include a link to your demo GIF in the README. It's sparks interest to see a demo of the app right away.
  - Include a note on how to get support (e.g. opening an issue).
2. Those interesting in running the app locally, and potentially helping you develop your dashboard
  - Potential contributors are interested in the above as well, but also need to know how they can install your app/dependencies and how to run it locally.
  - Include a link to the contributing file, where they can find out more if they are interested in making contributions to the repo.

### PR DESCRIPTION
**Pull request by SoloSynth1**
This PR addresses the improvements we need to work on the `README` file, as per Milestone 2's requirement.

Expand on the README file to be a welcoming place for anyone coming to your project for the first time. For your project, your README should cater to at least two groups of people:

- [x] Those interested in using your dashboard
  - [x] Include motivation behind your project and clearly explain what problem you are solving and why it is important.
  - [x] You do not have to include detailed usage instructions, just high level what they can do in your dashboard and the deployed link. [This is a good example](https://github.com/KirstieJane/STEMMRoleModels)
  - [x] Include a link to your demo GIF in the README. It's sparks interest to see a demo of the app right away.
  - [x] Include a note on how to get support (e.g. opening an issue).
- [x] Those interesting in running the app locally, and potentially helping you develop your dashboard
  - [x] Potential contributors are interested in the above as well, but also need to know how they can install your app/dependencies and how to run it locally.
  - [x] Include a link to the contributing file, where they can find out more if they are interested in making contributions to the repo.

Closes #54.

### CODE REVIEW

**Comment by SoloSynth1** [comment]
Marking this as a draft as there are unfinished items.

File: README.md [hunk 1]
```diff
@@ -14,47 +20,47 @@ Dr. Darling and our group are concerned about the alarming rise in national drug
...
+
+3. Install all necessary packages via `pip`:
+```bash
+pip install -r requirements.txt
 ```
-- To run the dashboard
+
+4. Run the dashboard:
 ```bash 
 python src/app.py
```
Comment (AlysenTownsley) [inline README.md:52]:
```suggestion
gunicorn src.app:server
```

**Review by AlysenTownsley — APPROVED** [review]
Looks good.

**Review by billwan96 — APPROVED** [review]
LGTM!

**Pull request merged**